ROLES.md

```diff
@@ -1,123 +1,88 @@
-    <td>Working Group</td>
+    <td>Working Group Lead</td>
+    <td>Responsible for the technical health and growth of their functional area or subproject</td>
   </tr>
+</table>
 
-  <tr>
-    <td>Working Group Execution Lead</td>
-    <td>
-        <p>Run their working group: Meetings, notes, roadmap, report</p>
-        <p>Responsible for the holistic health of the <i>working group</i></p>
-        <p>Work organization, planning, high-level execution</p>
-        <p>Triage incoming issues, set milestones, repo labels</p>
-        <p>Ensure all required skills are present within the working group</p>
-        <p>Mentor new contributors, project members, and approvers</p>
-    </td>
-    <td>Sponsored by the technical oversight committee as documented
-        <a href="./mechanics/WORKING-GROUP-PROCESSES.md">here</a>
-    </td>
-    <td>Write permissions on one or more repos allowing issues to be manipulated</td>
-    <td>Working Group</td>
-  </tr>
+Please note that repos in the knative-sandbox GitHub org are a reflection of
+the WG leads. These leads are free to use their judgement to set the bar for
+what is required to become an approver and/or lead for knative-sandbox repos
+they are responsible for.
 
-  <tr>
-    <td>Scribe</td>
-    <td>
-        <p>Ensure important information is represented in working group notes</p>
-        <p>Post WG recordings to team drive</p>
-    </td>
-    <td>Sponsored by a working group execution or technical lead.</td>
-    <td>Write permissions to team drive and team calendar</td>
-    <td>Working Group</td>
-  </tr>
+## Contributor
 
-</table>
+This role describes people who have just started contributing, or who
+contribute occasionally but don't participate in project governance or
+have defined responsibilities.
 
-Please note that repos in the knative-sandbox GitHub org are a reflection of the WG leads. These leads are free to use their judgement to set the bar for what is required to become an approver and/or owner for knative-sandbox repos they are responsible for.
+A Contributor contributes directly to the project and adds value to it.
+Contributions need not be code. People at the Contributor level may be new
+contributors, or they may only contribute occasionally.
+### Requirements
+
+Contributors should be familiar with the project and its processes, and should
+contribute to the project in one or more of the following ways.
+
+- Report and sometimes resolve issues
+- Occasionally submit PRs
+- Contribute to the documentation
+- Show up at meetings
+- Submit feedback on issues and PRs
+- Test releases and patches and submit reviews
+- Run or helps run events
+- Promote the project in public
 
 ## Member
 
-Established community members are expected to demonstrate their adherence to the
+A Member is an established Contributor who regularly participates in the
+project. Members have privileges in both project repositories and elections,
```

Comment:
I thought that elections were specifically limited to people who had 50 contributions in the last year, not to a "member level" (though it would be odd to have 50 contributions and not be an org member).

Reply:
I wondered about this. The wording comes from the CNCF template referenced in the PR description. I think being a Member should allow a contributor to participate in elections, and we should eliminate the 50 contribution requirement. "Contributions" are hard to quantify outside of actual GitHub activity, and we define "contributions" as including other activities, such as @nainaz being a presence at KubeCon and representing the project in a large public setting. How many "contributions" does that account for? In my opinion, being a Member should be enough.

Reply:
Our current elections rules also allow individuals to request an exception if they have non-GitHub contributions.  Note that contributions include issue and PR comments in addition to commits.

Reply:
Done: https://github.com/knative/community/issues/1350